wire-ios-data-model/Tests/Source/Utils/MockProteusProvider.swift

```diff
@@ -19,25 +19,23 @@
 import Foundation
 import WireDataModel
 
-@testable import WireDataModelSupport
+public class MockProteusProvider: ProteusProviding {
```

Comment:
I would prefer to move all files which belong to the same target to a unique directory tree.
For WireDataModelSupport I created a folder which is just named `Support`. Feel free to change the name.

<img width="521" alt="Screenshot 2023-12-12 at 10 40 11" src="https://github.com/wireapp/wire-ios/assets/15628584/78a0bf2b-031e-4520-9b8c-8b5bed45d58c">